### PR TITLE
README.rst: add instructions to run test

### PR DESCRIPTION
**Pull request by manu-chroma**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -40,6 +40,32 @@ Remember, if co-installing multiple CWL implementations then you need to
...
+
+-  Running conformance tests:
+
+The conformance tests are intended to test feature coverage of a CWL
+implementation. It uses the module “cwltool.cwltest” from the cwltool
+reference implementation.
+
+.. code:: bash
+
+      $ ./run_test.sh
```
Comment (mr-c) [inline README.rst]: We need to tell people to check out the CWL specs repository to get a copy of the conformance tests and `run_test.sh`

File: README.rst [hunk 2]
```diff
@@ -40,6 +40,32 @@ Remember, if co-installing multiple CWL implementations then you need to
...
 
+Running tests locally
+---------------------
+
+-  Running unnittests ``(/tests)``:
+
+.. code:: bash
+
+    pip install tox
+    tox # to run all tests
```
Comment (mr-c) [inline README.rst]: Developers might not have all the versions of Python installed that we test using `tox`, let's give them instructions on how to run the tests w/o tox

**Comment by manu-chroma** [comment]
I've made the relevant changes.

File: README.rst [hunk 3]
```diff
@@ -40,6 +40,23 @@ Remember, if co-installing multiple CWL implementations then you need to
...
+
+-  Running basic tests ``(/tests)``:
+
+.. code:: bash
+
+    python setup.py test
+
+-  Running the entire suite of CWL conformance tests:
+
+The GitHub repostiory for the CWL specifications contains a script that tests a CWL
```
Comment (manu-chroma) [inline README.rst]: minor typo here.

File: README.rst [hunk 4]
```diff
@@ -40,6 +40,23 @@ Remember, if co-installing multiple CWL implementations then you need to
...
+implementation against a wide array of valid CWL files using the [cwltest](https://github.com/common-workflow-language/cwltest)
```
Comment (manu-chroma) [inline README.rst]: you're using markdown formatting for inserting hyperlinks. I can add a commit to fix it for rst formatting.

**Comment by mr-c** [comment]
Jenkins, test this please

**Comment by mr-c** [comment]
Thank you @manu-chroma !

**Pull request merged**